src/JabbR-Core/Hubs/Chat.cs

```diff
@@ -186,7 +187,7 @@ public void UpdateActivity()
                 UpdateActivity(user, room);
             }
 
-            CheckStatus();
+            CheckStatus(testing);
```

Comment:
Why are we allowing out of band version checks? Seems like this is something that we can easily forget to leave in and could be configured via config settings on the server side to allow out of band clients (only if env.IsDevelopment)